### PR TITLE
load libpcrecpp before collada-dom, fixes #89

### LINKED ISSUE #89: urdf_to_collada:
I get the following errors and I can't find the error. The command used is 

rosrun collada_urdf urdf_to_collada ./input.urdf  ./out.dae

I/O error : No such file or directory
I/O error : No such file or directory
error : xmlNewTextWriterFilename : cannot open uri

My system is Ubuntu 14.04, 64bits, ROS Indigo. ROS Answer didn't help much :(

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

**Comment by davetcoleman** [comment]
wow, great fix!

File: collada_urdf/package.xml [hunk 1]
```diff
@@ -25,6 +25,7 @@
   <build_depend>collada_parser</build_depend>
   <build_depend>liburdfdom-dev</build_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
+  <build_depend>pcrecpp</build_depend>
```
Comment (wjwwood) [inline collada_urdf/package.xml:28]: Is `pcrecpp` different from `pcre` or `pcre-dev`?

https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L2619-L2632

I don't see `pcrecpp` in the `rosdep` database.

Comment (wjwwood) [inline collada_urdf/package.xml:28]: I'm gonna go with no, based on the stuff I've looked at, I'm going to change this to `pcre` and `pcre-dev`. @isucan

Comment (isucan) [inline collada_urdf/package.xml:28]: +1, thanks @wjwwood

**Pull request merged**